metrics.yaml

```diff
@@ -13,6 +13,23 @@
 ---
 $schema: moz://mozilla.org/schemas/glean/metrics/2-0-0
 
+study_enrollment:
+  id:
+    type: uuid
+    lifetime: user
+    send_in_pings:
+      - study-enrollment
+    description: |
+      The id of the Rally client. This is generated by
+      the Rally Core Add-on.
+    bugs:
+      - https://github.com/mozilla-rally/rally-study-01/issues/114
+    data_reviews:
+      - forthcoming.
+    notification_emails:
+      - hulmer@mozilla.com
+    expires: never
+
```

Comment:
I don't think you need this: you can add `study-enrollment` to the `send_in_pings` property of `rally.id` on line 39